tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/AmqpSendReceiveTest.java

```diff
@@ -53,38 +56,61 @@
 import org.apache.qpid.proton.engine.Sender;
 import org.jgroups.util.UUID;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * Test basic send and receive scenarios using only AMQP sender and receiver links.
  */
+@RunWith(Parameterized.class)
```

Comment:
This seems like an unnecessary change to this test.  The intent here is to test basic AMQP protocol support not to test the FQQN support in the broker which it appears is already done in the ProtonFullQualifiedNameTest.  I'd prefer if we could keep tests targeted and not try and make them test the entirety of the broker features in one monolithic test case as that makes it harder to maintain.  If you changed this one why didn't you change every single AMQP test in the same fashion?